exporters/prometheus/exporter.go

```diff
@@ -232,6 +245,55 @@ func getAttrs(attrs attribute.Set) ([]string, []string) {
 	return keys, values
 }
 
+func createTargetInfoMetric(res *resource.Resource) (*metricData, prometheus.Labels) {
+	if res == nil {
+		return nil, nil
+	}
+
+	var namespace, name, instance string
+	iter := res.Set().Iter()
+	for iter.Next() {
+		kv := iter.Attribute()
+		switch kv.Key {
+		case semconv.ServiceNameKey:
+			name = kv.Value.Emit()
+		case semconv.ServiceNamespaceKey:
+			namespace = kv.Value.Emit()
+		case semconv.ServiceInstanceIDKey:
+			instance = kv.Value.Emit()
+		}
+	}
+
+	if name == "" {
+		return nil, nil
+	}
+
+	constLabels := prometheus.Labels{
```

Comment:
We should not be adding constant labels to all metrics on the endpoint.

From the OM [spec](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#metricset):

`The same label name and value SHOULD NOT appear on every Metric within a MetricSet.`

Reply:
removed it.